src/lib/auth/oauth-client.mjs

```diff
@@ -52,64 +61,73 @@ class OAuthClient {
     return Buffer.from(randomBytes(20)).toString("base64url");
   }
 
+  _handleRedirect({ pathname, res }) {
```

Comment:
Prefix with `#` instead to make it private?

Reply:
We'd need to update the shared eslint configuration and that's outside the scope of this PR.

Reply:
Ah, bummer.